### PR TITLE
Feature/68 add scoreboard

### PR DESCRIPTION
**Pull request by pkulev**
**Closes issue(s):** [ #68 ]

**Description of the changes:**

  * What's new:
    * scoreboard file routines

  * Fixes:
    * fix ImportError in utils that was raised every time

  * Refactoring:
    * move PREFIX to tests.common

### CODE REVIEW

File: xoinvader/scoreboard.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+"""Scoreboard routines.
+
+Scoreboard file is CSV of username,score rows.
+If file doesn't exists it will be recreated from default stub.
```
Comment (vaartis) [inline xoinvader/scoreboard.py]: exists -> exist, don't need 's' if there's does already

File: xoinvader/scoreboard.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+DEFAULTS = [
+    ("most", 1700),
+    ("most", 1440),
+    ("most", 0),
+]
+"""Default scoreboard placeholder."""
+
+
+def ensure():
+    """Ensure that path and file exists, create default stub if not."""
```
Comment (vaartis) [inline xoinvader/scoreboard.py]: exists -> exist

File: xoinvader/scoreboard.py (hunk 1 above)
Comment (pkulev) [reply xoinvader/scoreboard.py]: Ah, I know it, typo.

File: xoinvader/scoreboard.py (hunk 2 above)
Comment (pkulev) [reply xoinvader/scoreboard.py]: Ok.

**Pull request merged**